tests/integration_tests/test_keras.py

```diff
@@ -9,8 +9,9 @@
 from optuna.testing.integration import DeterministicPruner
 
 
-def test_keras_pruning_callback():
-    # type: () -> None
+@pytest.mark.parametrize("interval, epochs", [(1, 1), (2, 1), (2, 2)])
+def test_keras_pruning_callback(interval, epochs):
```

Comment:
Just a nit, and we can fix it in another follow up PR.

For me, it is easier to understand what they are testing if we divide `test_keras_pruning_callback` and `test_keras_pruning_callback_with_interval`.

c.f. 
https://github.com/optuna/optuna/pull/1154/files#diff-58bac36a299f885bfb5a18600c4b6cb1R110